config/environments/development.rb

```diff
@@ -15,7 +15,7 @@
   config.consider_all_requests_local = true
 
   # Enable/disable caching. By default caching is disabled.
-  if Rails.root.join('tmp/caching-dev.txt').exist?
+  if Rails.root.join('path', 'to').exist?
```

Comment:
`Rails.root.join('tmp/caching-dev.txt').exist`is a default comand from rails 5. I changed it now but I am unsure it is correct (http://stackoverflow.com/questions/3724487/rails-root-directory-path)